rossum_api/clients/external_sync_client.py

```diff
@@ -1728,6 +1732,105 @@ def delete_hook(self, hook_id: int) -> None:
         """
         return self.internal_client.delete(Resource.Hook, hook_id)
 
+    # ##### RULES #####
+    def list_rules(
+        self, ordering: Sequence[RuleOrdering] = (), **filters: Any
+    ) -> Iterator[RuleType]:
+        """Retrieve all :class:`~rossum_api.models.rule.Rule` objects satisfying the specified filters.
+
+        Parameters
+        ----------
+        ordering
+            List of object names. Their URLs are used for sorting the results
+        filters
+            id: ID of a :class:`~rossum_api.models.rule.Rule`.
+
+            name: Name of a :class:`~rossum_api.models.rule.Rule`.
+
+            schema: ID of a :class:`~rossum_api.models.schema.Schema`.
+
+            rule_template: URL of the rule template the rule was created from.
+
+            organization: ID of a :class:`~rossum_api.models.organization.Organization`.
+
+        References
+        ----------
+        https://elis.rossum.ai/api/docs/#list-all-rules.
+
+        https://elis.rossum.ai/api/docs/#rule.
+        """
+        for r in self.internal_client.fetch_resources(Resource.Rule, ordering, **filters):
+            yield self._deserializer(Resource.Rule, r)
+
+    def retrieve_rule(self, rule_id: int) -> RuleType:
+        """Retrieve a single :class:`~rossum_api.models.rule.Rule` object.
+
+        Parameters
+        ----------
+        rule_id
+            ID of a rule to be retrieved.
+
+        References
+        ----------
+        https://elis.rossum.ai/api/docs/#retrieve-rule.
+
+        https://elis.rossum.ai/api/docs/#rule.
+        """
+        rule = self.internal_client.fetch_resource(Resource.Rule, rule_id)
+        return self._deserializer(Resource.Rule, rule)
+
+    def create_new_rule(self, data: dict[str, Any]) -> RuleType:
```

Comment:
Looking at the dict here, we should also think about custom serializer (not in this MR of course 😉).